mainWindow.py

```diff
@@ -151,18 +140,21 @@ def menuItems(self):
         HideShowButtons.triggered.connect(lambda: self.hideShowButns())
         viewMenu.addAction(HideShowButtons)
 
-
-        row = QAction("current row", self)
+        HelpButton = QAction("Manual", self)
         #no shortcut
-        row.triggered.connect(self.table_widget.printCurrentRow)
-        viewMenu.addAction(row)
+        HelpButton.triggered.connect(lambda: self.openHelp())
+        helpMenu.addAction(HelpButton)
+        
 
     def hideShowButns(self):        
         if self.control_buttons_widget.isVisible():
             self.control_buttons_widget.hide()
         else:
             self.control_buttons_widget.show()
 
+    def openHelp(self):
+        subprocess.Popen("file1.c",shell=True)
+
```

Comment:
file1.c will be opened with the users prefered application for .c programs. We can create a pdf from the readme and open that when ready.